.github/workflows/main_workflow.yml

```diff
@@ -213,19 +213,9 @@ jobs:
           docker pull trufflesuite/ganache:beta
 
           # build required images
-          pip install .
-          autonomy fetch --local --service valory/hello_world
 
           docker build deployments/Dockerfiles/tendermint/ -t valory/open-autonomy-tendermint:0.1.0 --file deployments/Dockerfiles/tendermint/Dockerfile
           docker build deployments/Dockerfiles/tendermint/ -t valory/open-autonomy-tendermint:1.0.0 --file deployments/Dockerfiles/tendermint/Dockerfile
-          
-          cd hello_world
```

Comment:
Ok but you need to build the new base image here?

Reply:
addressed in https://github.com/valory-xyz/open-autonomy/pull/1221/commits/bd5f6a2943481bfac13d3678769e6d735b716e53